client/packages/programs/src/JsonForms/common/components/Array/common.tsx

```diff
@@ -217,14 +219,13 @@ export const ArrayCommonComponent = (props: ArrayControlCustomProps) => {
               expanded={expandedItems[index]}
               onChange={() => handleToggle(index)}
               sx={{
-                mt: '0 !important',
-                mb: index === data.length - 1 ? '20px !important' : 1,
                 border: isError ? '1px solid red' : '',
               }}
             >
               <AccordionSummary
                 expandIcon={<ChevronDownIcon />}
                 sx={{
+                  height: '32',
```

Comment:
I made this array component a bit more compact. Previously it was hard to see that the items belong to the same array.